spec/abstractops.html

```diff
@@ -1428,12 +1427,15 @@ <h1>
       <dd>It parses the argument as an ISO 8601 string and returns the calendar identifier, or *undefined* if there is none.</dd>
     </dl>
     <emu-alg>
-      1. Let _parseResult_ be ParseText(StringToCodePoints(_isoString_), |TemporalCalendarString|).
+      1. Let _parseResult_ be ParseText(StringToCodePoints(_isoString_), |CalendarDateTime|).
       1. If _parseResult_ is a List of errors, throw a *RangeError* exception.
-      1. Let _id_ be the source text matched by the |CalendarName| Parse Node contained within _parseResult_, or an empty sequence of code points if not present.
-      1. If _id_ is empty, then
+      1. Let _calendar_ be the source text matched by |Calendar| Parse Node contained within _parseResult_, or an empty sequence of code points if not present.
+      1. If _calendar_ is not empty, return CodePointsToString(_calendar_).
+      1. Let _parseResult_ be ParseText(StringToCodePoints(_isoString_), |TemporalCalendarString|).
```

Comment:
This is setting an existing variable to a new value, so it should say "Set _parseResult_ to..."